src/demo/App.js

```diff
@@ -2,73 +2,88 @@
 import React from 'react';
 import Cytoscape from '../lib/components/Cytoscape.react.js';
 
-
 const elements = [
     {
         data: {id: 'one', label: 'Node 1'},
-        position: {x: 50, y: 50}
+        position: {x: 50, y: 50},
```

Comment:
you can actually just delete `src/demo` entirely, along with the `start` package script and the `webpack-serve` devDep. AFAICT nobody uses the standalone react demo anymore even during early component dev, we just dev in an actual Dash app. So likely as not the standalone demo is out-of-date or outright broken in one way or another.